### ISSUE TITLE
[BUG] Division by Zero in CPU calculation

### ISSUE THREAD

**Issue by nifisher** [aristocratos/bpytop#146]
**Describe the bug**

Division by zero.
Launched the program and it quit immediately with a division by zero error (see screenshot section)

**To Reproduce**

Launch the program

**Expected behavior**

not quit with an error

**Screenshots**

```
[webserver (1.0)]:: bpytop
Bpytop exited with errorcode (1). See /home/nifisher/.config/bpytop/error.log for more information!

[webserver (1.0)]:: cat /home/nifisher/.config/bpytop/error.log
16/10/20 (10:31:57) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2598, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 2834, in _draw
    CpuBox._draw_fg()
  File "/usr/bin/bpytop", line 1756, in _draw_fg
    out += (f'{THEME.inactive_fg} ⡀⡀⡀⡀⡀{Mv.l(5)}{THEME.gradient["temp"][min_max(cpu.cpu_temp[0][-1], 0, cpu.cpu_temp_crit) * 100 // cpu.cpu_temp_crit]}{Graphs.temps[0](None if cls.resized else cpu.cpu_temp[0][-1])}'
ZeroDivisionError: integer division or modulo by zero
16/10/20 (10:31:57) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 

[webserver (1.0)]:: uname -a
Linux webserver 4.14.180-1-ARCH #1 SMP PREEMPT Tue May 26 02:05:58 UTC 2020 armv7l GNU/Linux
```

**Info (please complete the following information):**
 - bpytop version: `bpytop -v`:  1.0.42
 - psutil version: `bpytop -v` (version 5.7.0 or above is required): 5.7.2
 - (Linux) Linux distribution and version: (see screenshot) Arch/armv7l
 - (OSX/FreeBSD) Os release version: n/a
 - Terminal used: bash
 - Font used: no idea offhand. doubt it's relevant.
 - Python version, `python3 --version` (version 3.6 or above is required): 3.8.6

**Additional context**

contents of `~/.config/bpytop/error.log`

```
16/10/20 (10:31:57) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2598, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 2834, in _draw
    CpuBox._draw_fg()
  File "/usr/bin/bpytop", line 1756, in _draw_fg
    out += (f'{THEME.inactive_fg} ⡀⡀⡀⡀⡀{Mv.l(5)}{THEME.gradient["temp"][min_max(cpu.cpu_temp[0][-1], 0, cpu.cpu_temp_crit) * 100 // cpu.cpu_temp_crit]}{Graphs.temps[0](None if cls.resized else cpu.cpu_temp[0][-1])}'
ZeroDivisionError: integer division or modulo by zero
16/10/20 (10:31:57) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 
```

**Comment by nifisher** [comment]
Correction on terminal: rxvt-unicode-minimal 9.22-11
And while I was looking things up, my font:
*font:    -windows-montecarlo-medium-r-normal-*-11-110-*-*-*-*-*-*
*boldFont: -windows-montecarlo-bold-r-normal-*-11-110-*-*-*-*-*-*


And to emphasize, this is not an x86 family cpu that it's running on.

**Comment by aristocratos** [comment]
@nifisher 
This looks like it's a issue with the cpu_temp_crit variable.

Could you run `python3 -c "import psutil; print(psutil.sensors_temperatures())"`

I'm gonna guess your "high" and "crit" temps for your cpu is either 0 or missing. Will take a look at it tomorrow and add a extra check if necessary.

**Comment by nifisher** [comment]
```
[webserver (1.0)]:: python3 -c "import psutil; print(psutil.sensors_temperatures())"

{'cpu0-thermal': [shwtemp(label='', current=58.0, high=0.115, critical=0.115)], 'cpu1-thermal': [shwtemp(label='', current=56.0, high=0.115, critical=0.115)], 'cpu2-thermal': [shwtemp(label='', current=59.0, high=0.115, critical=0.115)], 'cpu3-thermal': [shwtemp(label='', current=64.0, high=0.115, critical=0.115)], 'gpu-thermal': [shwtemp(label='', current=52.0, high=1.2e-07, critical=1.2e-07)]}
```

**Comment by nifisher** [comment]
bit more readable:
```
[webserver (1.0)]:: python3 -c "import psutil; import pprint; pp=pprint.PrettyPrinter(indent=3);pp.pprint(psutil.sensors_temperatures())"
{  'cpu0-thermal': [  shwtemp(label='', current=58.0, high=1.15e-10, critical=1.15e-10)],
   'cpu1-thermal': [  shwtemp(label='', current=56.0, high=1.15e-10, critical=1.15e-10)],
   'cpu2-thermal': [  shwtemp(label='', current=59.0, high=1.15e-10, critical=1.15e-10)],
   'cpu3-thermal': [  shwtemp(label='', current=64.0, high=1.15e-10, critical=1.15e-10)],
   'gpu-thermal': [  shwtemp(label='', current=52.0, high=0.00011999999999999999, critical=0.00011999999999999999)]}

```

**Comment by aristocratos** [comment]
@nifisher 
Yeah, those `high=0.115, critical=0.115` doesn't make much sense. Do you get the same "high" and "crit" if you run `sensors`?

**Comment by nifisher** [comment]
```
[webserver (1.0)]:: sensors
ads7846-spi-1-1
Adapter: SPI adapter
in0:           2.50 V  
in1:          10.00 V  
```

**Comment by aristocratos** [comment]
@nifisher 
Fixed in v1.0.43

**Comment by nifisher** [comment]
looks to work. thamks
